pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -101,6 +101,8 @@ const (
 	// RunCancelledByPipelineMsg indicates that the PipelineRun of which part this Run was
 	// has been cancelled.
 	RunCancelledByPipelineMsg RunSpecStatusMessage = "Run cancelled as the PipelineRun it belongs to has been cancelled."
+	// RunCancelledByPipelineTimeoutMsg indicates that the Run was cancelled because the PipelineRun running it timed out.
+	RunCancelledByPipelineTimeoutMsg RunSpecStatusMessage = "Run cancelled as the PipelineRun it belongs to has timed out."
```

Comment:
Is this status message visible to the user?

Reply:
It shows up as the condition message on the `TaskRun` or `Run`.

Reply:
thanks @abayer, please document this update - https://github.com/tektoncd/pipeline/blob/main/docs/taskruns.md#monitoring-execution-status

Reply:
Done.